core_backend/tests/api/test_manage_content.py

```diff
@@ -257,3 +263,21 @@ def test_auth_retrieve(
             headers={"Authorization": f"Bearer {access_token}"},
         )
         assert response.status_code == expected_status
+
+
+def test_convert_record_to_schema() -> None:
```

Comment:
I took inspiration from the qdrant upsert tests. But this test is the only test I found relevant. Let me know if I should add more.

Reply:
It's done already. I wanted to add new test cases for pgvector like it was done for qdrant (to test the payload upsert) but didn't think it was relevant since **sqlalchemy** handles the upsert.